src/pages/Anix/main.ts

```diff
@@ -0,0 +1,127 @@
+import { Animeworld } from '../Animeworld/main';
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Anix: pageInterface = {
+  domain: 'https://anix.to',
+  languages: ['English'],
+  name: 'Anix',
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return true;
+  },
+  sync: {
+    getTitle(url: string): string {
+      return j.$('h1.anime-name').text().trim();
+    },
+    getIdentifier(url: string): string {
+      const anime = url.split('/')[4];
+      const animeWords = anime.split('-');
+      return animeWords[animeWords.length - 1];
+    },
+    getOverviewUrl(url: string): string {
+      return utils.absoluteLink(
+        j.$('div.range-wrap > div.range > div > a').first().attr('href'),
+        Anix.domain,
+      );
+    },
+    getEpisode(url: string): number {
+      return parseInt(j.$('div.range-wrap > div.range > div > a.active').text());
+    },
+    nextEpUrl(url: string): string | undefined {
+      return j
+        .$('div.range-wrap > div.range > div > a.active')
+        .parent('div')
+        .next()
+        .find('a')
+        .attr('href');
+    },
+    uiSelector(selector) {
+      j.$('#ani-player-section').before(j.html(selector));
+    },
+    readerConfig: undefined,
+  },
+  overview: {
+    getTitle(url) {
+      return '';
+    },
+    getIdentifier(url) {
+      return '';
+    },
+    uiSelector(selector) {
+      // no Ui
+    },
+    list: {
+      offsetHandler: false,
+      elementsSelector() {
+        return j.$(
+          'div.ani-episode > div.range-wrap > div.range > div > a:not([style*="display: none"])',
```

Comment:
Too specifc

Reply:
Now is `#ani-episode a:not([style*="display: none"])`